autotst/reaction.py

```diff
@@ -149,7 +205,7 @@ def distance_data(self):
         return self._distance_data
 
     @classmethod
-    def load_databases(self, force_reload=False):
+    def load_databases(self, rmg_database_path=None, force_reload=False):
```

Comment:
I like the addition of a path (especially for developers using different databases). Can you add comments to the method describing this as a variable?